vllm/model_executor/layers/quantization/awq.py

```diff
@@ -2,7 +2,7 @@
 
 import torch
 from torch.nn.parameter import Parameter
-
+import intel_extension_for_pytorch
```

Comment:
Note that the IPEX import should be lazy until actually required for weight_only_qlinear_prepack_int4